blackduck/Client.py

```diff
@@ -195,11 +199,55 @@ def get_metadata(self, name, parent=None, **kwargs):
         kwargs['params'] = {'limit': 1}
         return self.get_resource(name, parent, items=False, **kwargs)
 
-    def print_methods(self):
-        import inspect
-        for fn in inspect.getmembers(self, predicate=inspect.ismember):
-            print(fn[0])
+    def checked_request(self, method, url, name='', **kwargs):
```

Comment:
As this is now public, I renamed it from _request to checked_request. I am open to other names.